compiler/ballerina-backend-jvm/src/main/ballerina/compiler_backend_jvm/jvm_method_gen.bal

```diff
@@ -819,7 +819,8 @@ function generateMainMethod(bir:Function userMainFunc, jvm:ClassWriter cw, bir:P
 
     mv.visitTypeInsn(NEW, SCHEDULER);
     mv.visitInsn(DUP);
-    mv.visitMethodInsn(INVOKESPECIAL, SCHEDULER, "<init>", "()V", false);
+    mv.visitInsn(ICONST_4);
```

Comment:
Don't we need to have some sort of logic or a way to configure number of threads? How about codegening this to at-least get it from a utility method for the moment.

Reply:
Will refactor in next pr. Created #15307 to track this.